packages/story-editor/src/components/checklist/checks/pageBackgroundLowTextContrast/component.js

```diff
@@ -34,7 +34,7 @@ import { getVisibleThumbnails, ThumbnailPagePreview } from '../../utils';
 import { ACCESSIBILITY_COPY } from '../../constants';
 import { useRegisterCheck } from '../../countContext';
 import { useIsChecklistMounted } from '../../popupMountedContext';
-import { pageBackgroundTextLowContrast } from './check';
+import { getFailingPages } from './check';
```

Comment:
nit: could this be renamed as like `getPagesWithFailedContrast` or something that's clearer since it's now getting used outside of `/pageBackgroundLowTextContrast` which previously provided more context.